src/time_integration/methods_SSP.jl

```diff
@@ -0,0 +1,239 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+
+# Abstract base type for time integration schemes of explicit strong stability-preserving (SSP)
+# Runge-Kutta (RK) methods. They are high-order time discretizations that guarantee the TVD property.
+abstract type SimpleAlgorithmSSP end
+
+
+"""
+    SimpleSSPRK33(; stage_callbacks=(APosterioriLimiter(),))
+
+The third-order SSP Runge-Kutta method of Shu and Osher.
+
+## References
+
+- Shu, Osher (1988)
+  "Efficient Implementation of Essentially Non-oscillatory Shock-Capturing Schemes" (Eq. 2.18)
+  [DOI: 10.1016/0021-9991(88)90177-5](https://doi.org/10.1016/0021-9991(88)90177-5)
+
+!!! warning "Experimental implementation"
+    This is an experimental feature and may change in future releases.
```

Comment:
Required?

Reply:
When we will add the MCL approach in the future, it becomes important trying to solve [this issue](https://github.com/trixi-framework/Trixi.jl/pull/1476#discussion_r1215356222) (since then the MCL would work with time integration methods of OrdinaryDiffEq). Until then, I think it's a good idea to keep the experimental warning.